server/monitor.go

```diff
@@ -2357,15 +2357,15 @@ func (s *Server) Jsz(opts *JSzOptions) (*JSInfo, error) {
 		js, cc := s.getJetStreamCluster()
 		if js == nil {
 			// Ignore
-			return nil, ErrJetStreamNotEnabled
+			return nil, fmt.Errorf("%w: no cluster", skipZreq)
```

Comment:
I do it this way as I didnt want to change the behavior for HTTP requests - I dont know what we want to do exactly when making a request over HTTP that isnt for us?

Anyway now it does:

```
# curl "0:8222/jsz?leader-only=true"
filtered response: not leader
```